.dotnet/src/Custom/Chat/StreamingChatCompletionUpdate.cs

```diff
@@ -42,6 +43,9 @@ public partial class StreamingChatCompletionUpdate
     [CodeGenMember("Usage")]
     public ChatTokenUsage Usage { get; }
 
+    [CodeGenMember("ServiceTier")]
+    internal InternalCreateChatCompletionStreamResponseServiceTier? _serviceTier;
```

Comment:
I recommend adding the CUSTOM comment and keeping these as properties instead of fields, so that they look exactly like the generated code except for the actual part that was customized (in this case, the access modifier from `public` to `internal`).

Reply:
Sure; no strong opinion from me. Updated.